src/clarity/iconography/demo/icon-orientation.demo.html

```diff
@@ -4,17 +4,39 @@
   ~ The full license information can be found in LICENSE in the root directory of this project.
   -->
 
+<h6>A. Set the icon orientation in the <code>shape</code> attribute</h6>
+
+<span class="clr-example">
+    <clr-icon shape="info-circle up"></clr-icon>
+    <clr-icon shape="info-circle right"></clr-icon>
+    <clr-icon shape="info-circle down"></clr-icon>
+    <clr-icon shape="info-circle left"></clr-icon>
```

Comment:
This might sound silly, but do you mind using an actual directional icon for this example? Because with something like an info circle, what's "left" to you might be "right" to someone else, and we're going to get issues filed for this. Arrows leave no space for discussion.